src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,156 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static Task[] tasks = new Task[100];
+    public static int taskCount = 0;
+
+    public static void printGreeting() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+    }
+
+    public static void printConfused() {
+        System.out.println("Could you say that again?");
+    }
+
+    public static void printExit() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void printGotIt() {
+        System.out.println("Got it. I've added this task:");
+    }
+
+    public static void printTaskCount() {
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+    }
+
+    public static void printTodoException() {
+        System.out.println("The description of todo cannot be empty.");
+    }
+
+    public static void printDeadlineException() {
+        System.out.println("The description of deadline cannot be empty.");
+    }
+
+    public static void printEventException() {
+        System.out.println("The description of event cannot be empty.");
+    }
+
+    public static void printDoneException() {
+        System.out.println("The description of done cannot be empty.");
+    }
+
+    public static void printTaskTypeResponse() {
+        //printing different responses depending if its duke.Todo/duke.Deadline/duke.Event
+        printGotIt();
+        tasks[taskCount - 1].printTask();
+        printTaskCount();
+    }
+
+    public static void printList() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i += 1) {
+            System.out.print((i + 1) + ".");
+            tasks[i].printTask();
+        }
+    }
+
+    public static void taskDone(String line) throws DoneException {
```

Comment:
Perhaps you could rephrase the method name here so that it starts with a verb according to https://se-education.org/guides/conventions/java/index.html#naming.